api4/team.go

```diff
@@ -1200,8 +1200,11 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 	if graceful {
 		cloudUserLimit := *c.App.Config().ExperimentalSettings.CloudUserLimit
 		var invitesOverLimit []*model.EmailInviteWithError
-		if cloudUserLimit > 0 && c.IsSystemAdmin() {
-			emailList, invitesOverLimit, _ = c.App.GetErrorListForEmailsOverLimit(emailList, cloudUserLimit)
+		if *c.App.Srv().License().Features.Cloud && cloudUserLimit > 0 && c.IsSystemAdmin() {
+			subscription, _ := c.App.Cloud().GetSubscription()
```

Comment:
We shouldn't avoid check or act in case of errors. In this case, if we don't want to react to the error because is nil we should at least log it so we know what is happening

Reply:
Meant to add a check but forgot after testing. Added one to check for the error being nil